gulpfile.js

```diff
@@ -70,6 +70,34 @@ const cssLoaderPlugin = {
 	},
 };
 
+gulp.task('ext:lint-staged', async (done) => {
```

Comment:
Odd that lint-staged affected this... and yet packaging the vsix locally was fine.

Can you add a comment on this gulp task explaining why this ended up having to be done manually rather than using the lint-staged package for posterity?